src/VS4Mac.TextGenerator/Services/TextGeneratorService.cs

```diff
@@ -20,8 +21,10 @@ public string[] GenerateWords(int numberOfWords)
 
         public string[] GenerateCharacters(int numberOfChars)
         {
-            string[] words = _generator.GenerateCharacters(numberOfChars);
-
+            //First three characters are \n or \r
+            var words = _generator.GenerateCharacters(numberOfChars + 3);
```

Comment:
Does this always happen?

Reply:
That seems..
https://github.com/alexcpendleton/NLipsum/blob/4c158dc577b31f3349199e096b91ea06101b54c4/NLipsum.Core/LipsumGenerator.cs#L158-L177
Maybe I should add a test in case the NLipsum nuget is updated in the future and this changes.
```suggestion
        [Test]
        public void FirstGenerateCharactersAreAlwaysTheSameTest()
        {
            var lipsumGenerator = new LipsumGenerator();
            var words = lipsumGenerator.GenerateCharacters(3);
            var characters = words[0];

            Assert.AreEqual(characters[0], '\r');
            Assert.AreEqual(characters[1], '\n');
            Assert.AreEqual(characters[2], '\t');           
        }